dashboard/app/assets/javascripts/angular/testExecutor.service.js

```diff
@@ -25,7 +25,7 @@
           method: 'DELETE'
         }
       });
-      var testExecUpdateIntervalPeriod = 5; // seconds
+      var testExecUpdateIntervalPeriod = 1; // seconds
```

Comment:
It sounds like 1 second is too low. How much data is it retrieving each time? Is each response fairly lightweight? Will the amount of data it's preparing and returning with each request become very burdensome if/when this is in heavy use?